### PR TITLE
Add account management category

### PR DESCRIPTION
**Pull request by Mrtenz**
This adds a new category "Account Management," which can be used by account management Snaps.

### CODE REVIEW

File: src/features/filter/store.test.ts [hunk 1]
```diff
@@ -125,33 +125,28 @@ describe('filterSlice', () => {
...
+      expect(newState.categories).toContain(
         RegistrySnapCategory.TransactionInsights,
-      ]);
+      );
     });
 
     it('sets the categories to all if all categories are toggled off', () => {
-      const state = filterSlice.reducer(
-        filterSlice.reducer(
+      const state = Object.values(RegistrySnapCategory).reduce(
```
Comment (Mrtenz) [inline src/features/filter/store.test.ts:143]: I refactored this slightly, so the test doesn't break if we add a new category.

File: src/constants.ts [hunk 2]
```diff
@@ -36,6 +41,11 @@ export const SNAP_CATEGORY_LINKS: Record<
     linkText: MessageDescriptor;
   }
 > = {
+  [RegistrySnapCategory.AccountManagement]: {
+    header: defineMessage`Manage Your Accounts`,
```
Comment (FrederikBolding) [inline src/constants.ts:45]: I think this is still TBD

File: src/features/snap/components/Data.tsx [hunk 3]
```diff
@@ -1,22 +1,35 @@
...
+        color={warning ? 'warning.default' : 'text.alternative'}
+        fontWeight="medium"
+        fontSize="sm"
+        textTransform="uppercase"
+      >
+        {label}
+      </Text>
+      {warning && (
+        <Tooltip label={warning} placement="auto-end">
+          <WarningIcon width="20px" fill="warning.default" />
```
Comment (FrederikBolding) [inline src/features/snap/components/Data.tsx:29]: `warning.default` is more orange in light mode. Is that intentional?

image

cc @eriknson

File: src/components/icons/add-user.svg
Comment (FrederikBolding) [inline src/components/icons/add-user.svg:1]: This icon looks broken

File: src/components/FooterAccountManagementTerms.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { Box, Text } from '@chakra-ui/react';
+import { Trans } from '@lingui/macro';
+import type { FunctionComponent } from 'react';
+
+import { FooterTerms } from './FooterTerms';
+
+export const FooterAccountManagementTerms: FunctionComponent = () => (
```
Comment (FrederikBolding) [inline src/components/FooterAccountManagementTerms.tsx:7]: Confirm whether we also need to show the existing terms at the bottom. We may need to

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: src/components/FooterAccountManagementTerms.tsx (hunk 4 above)
Comment (Mrtenz) [reply src/components/FooterAccountManagementTerms.tsx:7]: This includes the existing terms.

File: src/features/snap/components/Data.tsx (hunk 3 above)
Comment (Mrtenz) [reply src/features/snap/components/Data.tsx:29]: Yeah, this is the DS warning color.

File: src/constants.ts (hunk 2 above)
Comment (Mrtenz) [reply src/constants.ts:45]: Yep. I just put something there for now.

File: src/components/FooterAccountManagementTerms.tsx (hunk 4 above)
Comment (FrederikBolding) [reply src/components/FooterAccountManagementTerms.tsx:7]: yes nvm I'm blind

Comment (FrederikBolding) [reply src/components/FooterAccountManagementTerms.tsx:7]: Somehow I missed that 😓

**Review by FrederikBolding — DISMISSED** [review]

File: src/pages/snap/{Snap.location}/{Snap.slug}.tsx [hunk 5]
```diff
@@ -200,7 +201,9 @@ export const query = graphql`
         contact
         faq
         knowledgeBase
+        keyRecovery
```
Comment (FrederikBolding) [inline src/pages/snap/{Snap.location}/{Snap.slug}.tsx:204]: We should probably support `additionalSourceCode` in this PR too

**Review by FrederikBolding — DISMISSED** [review]

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**